dev_mode/webpack.config.js

```diff
@@ -222,7 +222,7 @@ module.exports = [
     },
     output: {
       path: plib.resolve(buildDir),
-      publicPath: 'static/lab/',
+      publicPath: '/static/lab/',
```

Comment:
Is that additional `/` needed? It maybe a rest of the time when the workspace loading were temporary no more working?

Reply:
Yes it is possible, will rebase!